common/app/common/configuration.scala

```diff
@@ -363,6 +363,8 @@ class GuardianConfiguration(val application: String, val webappConfDirectory: St
     lazy val pandomainHost = playConfiguration.getMandatoryStringFromStage("faciatool.pandomain.host")
     lazy val pandomainDomain = playConfiguration.getMandatoryStringFromStage("faciatool.pandomain.domain")
     lazy val pandomainService = playConfiguration.getMandatoryStringFromStage("faciatool.pandomain.service")
+    lazy val pandomainRoleArn= playConfiguration.getMandatoryStringFromStage("faciatool.pandomain.roleArn")
```

Comment:
*if* you're completely removing the secret then the config properties should come out too. As I mentioned it might be worth leaving them in the chain, in which case perhaps they should be optional properties (line 368)